scripts/xplaneUDP.py

```diff
@@ -0,0 +1,457 @@
+# Adapted from below, which was meant for receiving data
+# https://github.com/charlylima/XPlaneUDP/blob/master/XPlaneUdp.py
+
+# Method adapted to send data to XPlane instead of receive
+
+import socket
+import struct
+import binascii
+import pickle
+import os
+import numpy as np
+
+import sharpy.utils.solver_interface as solver_interface
+import sharpy.postproc
+from sharpy.utils.algebra import quat2euler
+
+import matplotlib.pyplot as plt
```

Comment:
Do you need matplolib? We do not include matplotlib in our environment that is used for testing (`environment_minimal`) so we tend to wrap any matplotlib import around something like:
```suggestion
try:
    import matplotlib.pyplot as plt
except ModuleNotFoundError:
    with_matplotlib = False
else:
   with_matplotlib = True
```

and then preceed any matplotlib calls you make with an `if with_matplotlib:` (or something along those lines)